osu.Game/Screens/OnlinePlay/Multiplayer/MultiplayerRoomManager.cs

```diff
@@ -34,10 +34,8 @@ protected override void LoadComplete()
             base.LoadComplete();
 
             isConnected.BindTo(multiplayerClient.IsConnected);
-            isConnected.BindValueChanged(_ => Schedule(updatePolling));
-            JoinedRoom.BindValueChanged(_ => updatePolling());
-
-            updatePolling();
+            isConnected.BindValueChanged(_ => Scheduler.AddOnce(updatePolling));
+            JoinedRoom.BindValueChanged(_ => Scheduler.AddOnce(updatePolling), true);
```

Comment:
I understand the schedule on `isConnected`, but what's the schedule on `JoinedRoom` for?

Reply:
That was just to benefit from `AddOnce` (ensuring minimal calls to `updatePolling`). Generally if using `AddOnce` on a call, I would want to see it on every call.

Reply:
I actually don't know if that's even going to work, because it's a method group (need to test). Besides that, I bring this up because I thought there was a reason for not scheduling this, like setting `allowPolling` too late resulting in missing the initial poll.

Reply:
No, no reason. Just standardising.

If this `AddOnce `call isn't working then none are. Method groups is the only way `AddOnce` should be used (inline delegates will not work unless they contain no captures).